app/views/posts/index.html.erb

```diff
@@ -4,11 +4,46 @@
   <title>User Posts</title>
 </head>
 <body>
-  <h1>User Posts</h1>
-  <p>User ID: <%= @user.id %></p>
-  <!-- Display user posts here -->
-  <% @posts.each do |post| %>
-    <p><%= post.title %></p>
-  <% end %>
+  <div class="user-container ">
+    <div class="users flex-spaced">
+      <div class="user-photo bordered">Photo</div>
+      <div class="details bordered flex-spaced">
+          <div class="user-name"><%= @user.name %></div>
+          <div class="post-nums">Number of posts: <%= @user.posts.count %></div>
+      </div>
+    </div>
+    <% @user.posts.each_with_index do |post, index| %>
+      <div class="post-comment">
+        <a href="<%= user_post_path(@user, post) %>" class="bio bordered flex-c">
```

Comment:
- [ ] For best practice I think it would be nice to use the [rails erb link syntax](https://www.rubyguides.com/2019/05/rails-link_to-method/) instead of an `<a herf=""></a>` tag. hint.

```suggestion
        <%= link_to user_post_path(@user, post)  class: "bio bordered flex-c" do %>
                 .......
        <% end  %>
```